WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -40346,13 +40365,22 @@ e_latin_empire = {
 	capital = 496 # Byzantion
 	
 	short_name = yes
+	dynasty_title_names = no
 	
 	
 	allow = {
 		OR = {
+			has_romance_culture_trigger = yes
+			culture_group = latin
 			religion = catholic
 			is_heresy_of = catholic
 		}
+
+		NOT = {
+			culture = greek
+			culture = roman
+		}
+			
```

Comment:
Because it is better to not change it all, that way the root of the problem no longer exists.